tests/scripts/github-action-helper.sh

```diff
@@ -121,7 +121,7 @@ function build_rook() {
   tests/scripts/validate_modified_files.sh build
   docker images
   if [[ "$build_type" == "build" ]]; then
```

Comment:
@subhamkrai This is working since the integration tests are not run in the case where `build.all` is used for publishing the build or building on linux.